localization/languages/pt-BR.json

```diff
@@ -194,10 +192,10 @@
         /* PDF Viewer */
         "PDFPageCounter": {"unsafeHTML": "página <input type='text'/> de <span id='total'></span>"},
         /* Download manager */
-        "downloadCancel": null, //missing translation
-        "downloadStateCompleted": null, //missing translation
-        "downloadStateFailed": null, //missing translation
+        "downloadCancel": "Download cancelado",
+        "downloadStateCompleted": "Download concluído",
+        "downloadStateFailed": "Download falhou",
```

Comment:
Sorry I missed this before, but these should just be "cancel", "completed", "failed".

(also, "cancel" is a verb in this case, as in "cancel this download")